app/App.tsx

```diff
@@ -13,6 +13,8 @@ import {
   ThemeProvider,
   toastConfig,
   TourProvider,
+  ErrorBoundaryWrapper,
+  bifoldLoggerInstance,
```

Comment:
In this project we should use the BCLogger from utils/logger. It's our RemoteLogger so it will include the proper reporting functionality